src/node.cc

```diff
@@ -3786,6 +3790,14 @@ static void ParseArgs(int* argc,
     index += args_consumed;
   }
 
+#if HAVE_OPENSSL
+  if (use_openssl_ca && use_bundled_ca) {
+    fprintf(stderr,
+        "--use-openssl-ca or --use-bundled-ca can be used, not both\n");
```

Comment:
Can you line up the arguments?  There should probably be an "either" in front of the error message and we prefix it with argv[0] in most cases.